velox/connectors/hive/HiveDataSink.cpp

```diff
@@ -735,7 +735,12 @@ std::pair<std::string, std::string> HiveDataSink::getWriterFileNames(
   const std::string writeFileName = isCommitRequired()
       ? fmt::format(".tmp.velox.{}_{}", targetFileName, makeUuid())
       : targetFileName;
-  return {targetFileName, writeFileName};
+  if (insertTableHandle_->tableStorageFormat() ==
+      dwio::common::FileFormat::PARQUET) {
+    return {targetFileName, fmt::format("{}{}", writeFileName, ".parquet")};
+  } else {
```

Comment:
nit: this else is not necessary.
@JkSelf, can we verify this change in the TableWriteTest.cpp? Some of them have Parquet format support. Thanks.

Reply:
A test also helps prevent breaking this change in the future.